app-frontend/src/containers/AuthPage/LogInContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import useHttp from '../../hooks/http.hook';
+import LogIn from '../../components/AuthPage/LogIn';
+import loginUser from '../../store/actions/authorizeUser/login';
+
+type LoginForm = {
+  email: string,
+  password: string,
+};
+
+const LogInContainer = function LogInContainer() {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const [error, setError] = useState<string>('');
+  const {
+    loading,
+  } = useHttp();
+  const [form, setForm] = useState<LoginForm>({
+    email: '', password: '',
+  });
+
+  const authorizationHandler = async (): Promise<void> => {
+    try {
+      dispatch(loginUser({ ...form }));
+      navigate('/');
```

Comment:
what if loginUser will fail? in this case, you'll just redirect user to '/'. But it is bad behavior, you should show an error message instead of redirection.